### DISCUSSION TITLE
Moving configs from Docker Desktop to Docker Ubuntu

### DISCUSSION THREAD

**Discussion by rwskinner** [jasonacox/Powerwall-Dashboard#539] [General]
Deleted

**Comment by jasonacox** [comment]
Did you get it working?

**Comment by rwskinner** [comment]
I went ahead and ran the setup script and set it up from scratch.  I still have it running under Docker Desktop so I don't get gaps in the DB.
I shut down both containers  (Docker Desktop and then the Ubuntu docker container) then I copied the influxdb and the graphana folders over to the new Ubuntu machine, but when I restarted the new Ubuntu Powerwall container, it only showed data from today and not the historic data, like it didn't actually see the database.  I thought maybe I need to follow the backup procedure although I sort of did, I just didn't compress them before moving.

**Reply by jasonacox** [reply]
That should work. You need to make sure the influxdb container is stopped. I suspect some of the files are not copying if you didn't use the archive method to preserve the user permissions.

```bash
# Stop influxdb
docker stop influxdb

# copy over the file into the "influxdb" folder

# Make sure the files have correct user and permission

# Start
./compose-dash.sh up -d
```

**Reply by rwskinner** [reply]
just the meta.db file right?

**Reply by jasonacox** [reply]
No, you will need the entire `data` folder (which is actually where the data is located) as well as the meta.db.

**Reply by rwskinner** [reply]
Can't copy the entire folder, even as root - access denied.  I did the backup script, then moved the tar over the the other machine and extracted it following the instructions.  Now, a blank dashboard with errors everywhere, invalid gateway.  I guess I nuked it.

**Reply by jasonacox** [reply]
```
docker logs influxdb
```

**Reply by rwskinner** [reply]
I finally got it.  I just had to follow the restore directions properly.

Thanks.

**Reply by jasonacox** [reply]
Good job!
